doc/c64.sgml

```diff
@@ -216,6 +250,9 @@ configuration.
   palette of the 16 C64 colors).
 </descrip><p>
 
+Note that the graphics drivers are incompatible with the
+<tt/c64-ram.emd (c64_ram_emd)/ extended memory driver and the
+ <tt/c64-soft80.o/ software 80 columns conio driver.
```

Comment:
Just for curiosity: Is the way the TGI driver programs the VIC incompatible with the way soft80 does? Or would it - at least conceptually - be possible to use soft80 and TGI together?